summingbird-online/src/main/scala/com/twitter/summingbird/planner/OnlinePlan.scala

```diff
@@ -146,7 +159,7 @@ class OnlinePlan[P <: Platform[P], V](tail: Producer[P, V]) {
            * summer (to handle map-side aggregation). This check is here to prevent us from merging
```

Comment:
The elimination of FlatMapNode only happens when the SourceNode output is fed to SummerNode. All other cases remain unchanged.
I will correct the comment accordingly.